back/app/serializers/web_api/v1/custom_field_serializer.rb

```diff
@@ -20,6 +20,10 @@ class WebApi::V1::CustomFieldSerializer < WebApi::V1::BaseSerializer
     object.input_type == 'page'
   }
 
+  attribute :dropdown_layout, if: proc { |object, _params|
+    %w[multiselect select].include?(object.input_type)
+  }
+
```

Comment:
I think we should add a couple of tests to [custom_field_serializer_spec.rb](https://github.com/CitizenLabDotCo/citizenlab/blob/master/back/spec/serializers/web_api/v1/custom_field_serializer_spec.rb).

One for each `CustomField.input_type` we expect to have this attribute serialized.

e.g.
```
context 'select field' do
...
it 'includes the dropdown_layout attribute' do
...
etc.
```